controllers/bsl_test.go

```diff
@@ -1174,6 +1174,201 @@ func TestDPAReconciler_ValidateBackupStorageLocations(t *testing.T) {
 			want:    true,
 			wantErr: false,
 		},
+		{
+			name: "BSL without config section for aws provider and default backupImages is true behaviour",
```

Comment:
NIT: behavior

Reply:
Done, updated (not a typo according to UK english though :) )